tests/nested/manual/muinstaller-real/task.yaml

```diff
@@ -328,8 +332,13 @@ execute: |
 
   # Make sure there is a change in assets to update
   sed -i 's/This program cannot be run in DOS mode/This program cannot be run in XXX mode/' pc-gadget/grubx64.efi
+
   # Bump edition of ubuntu-{seed,boot} content to get grub updated in both partitions
-  yq -i '(.volumes.pc.structure[] | select(.role == "system-seed-null") | .update.edition) |= . + 1' pc-gadget/meta/gadget.yaml
+  if [ "$SEED_PARTITION" = 'true' ]; then
+      yq -i '(.volumes.pc.structure[] | select(.role == "system-seed") | .update.edition) |= . + 1' pc-gadget/meta/gadget.yaml
```

Comment:
maybe a variable could be used to avoid repeating the almost identical yq command